From 75b1b2284aed35ee053e30b310a89ff8a52b8b73 Mon Sep 17 00:00:00 2001
From: David Mercado <david.mercado@windriver.com>
Date: Tue, 19 Nov 2013 11:52:55 -0500
Subject: [PATCH 130/175] LSI AXM55XX: DT "compatible" property in the form <manufacturer>,<model>

git.yoctoproject.org/git/linux-yocto-3.10
commit e9b4daf1f7548f47f1f1df6886cc46cb4b4bfe8e standard/axxia/base.

Change the "compatible" property in axm55xx.dts for USB and FEMAC to
the form <manufacturer>,<model>. Update "compatible" field in LSI
femac driver as well.

Signed-off-by: David Mercado <david.mercado@windriver.com>
---
 arch/arm/boot/dts/axm55xx.dts          |    4 ++--
 drivers/net/ethernet/lsi/lsi_acp_net.c |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/axm55xx.dts b/arch/arm/boot/dts/axm55xx.dts
index 88f10ea..896cb7f 100644
--- a/arch/arm/boot/dts/axm55xx.dts
+++ b/arch/arm/boot/dts/axm55xx.dts
@@ -250,7 +250,7 @@
 	};
 
 	axxia_femac0: femac@0x2010120000 {
-		compatible = "acp-femac";
+		compatible = "lsi,acp-femac";
 		device_type = "network";
 		reg = <0x20 0x10120000 0 0x1000>,
 		      <0x20 0x10121000 0 0x1000>,
@@ -332,7 +332,7 @@
 
 	USB0: usb@004a4000 {
 		device_type = "usb";
-		compatible = "acp-usb";
+		compatible = "lsi,acp-usb";
 		enabled = <0>;
 		reg = <0x20 0x10140000 0x0 0020000>,
 		      <0x20 0x10094000 0x0 0002000>;
diff --git a/drivers/net/ethernet/lsi/lsi_acp_net.c b/drivers/net/ethernet/lsi/lsi_acp_net.c
index bfb91ad..188c196 100644
--- a/drivers/net/ethernet/lsi/lsi_acp_net.c
+++ b/drivers/net/ethernet/lsi/lsi_acp_net.c
@@ -2143,7 +2143,7 @@ static int __devexit appnic_drv_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id appnic_dt_ids[] = {
-	{ .compatible = "acp-femac", }
+	{ .compatible = "lsi,acp-femac", }
 };
 MODULE_DEVICE_TABLE(of, appnic_dt_ids);
 
-- 
1.7.1

